### ISSUE TITLE
[BE] The generated ticket conforms to the correct format

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#33]
Expansion to #19 

The output from #19 conforms to the template as a JSON file

## Acceptance criteria
- [ ] the generated ticket categories match the template
- [ ] the generated ticket is in JSON format

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
